charts/agents-nuxt-service/values.yaml

```diff
@@ -0,0 +1,60 @@
+# Default values for agents-graphql-service.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from k8s-at-home common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+controller:
+  # -- Number of desired pods.
+  # We use 3 minimum to assure no outage durring rollout/preemtible node restarts
+  replicas: 3
+
+image:
+  # -- image repository
+  repository: "eu.gcr.io/logical-sled-220910/strg-at/agents-nuxt-service"
+  # -- image tag
+  tag: ""
+  # -- image pull policy
+  # pullPolicy: IfNotPresent
+
+securityContext:
+  capabilities:
+    drop:
+      # -- drop all privileges as we dont need them
+      - ALL
+  # -- set root fs to read only for security reasons
+  readOnlyRootFilesystem: true
+  # -- do not allow privilege escalation for security reasons
+  allowPrivilegeEscalation: false
+  # -- do not run as root for security reasons
+  runAsNonRoot: true
+  # -- run as user with <id>
+  runAsUser: 1000
+
+env:
+  # -- The timezone in the container
+  TZ: UTC
+  # -- The application port
+  PORT: "3000"
+  # -- The default node environment
+  NODE_ENV: "production"
+  # Url for connection to graphql
+  GRAPHQL_URL: "http://localhost:4000"
+  # Host for keycloak authentication
+  KEYCLOAK_HOST: "http://localhost:8180"
```

Comment:
This we can clarify, I can set it empty, but not sure if it will work.

Reply:
@tyriis  this check could be done in job as variables was taken from outside, but on my scripts that variables are define inside, so in case I want to add this kind of check, where should I add them? At end of script? 
Something like this?
```
{{- if eq .env.KEYCLOAK_HOST  "" }}
{{- fail "value for  .env.KEYCLOAK_HOST must be set" }}
{{- end }}
```

Reply:
@jazzlyn  variables are now set as empty: 
https://github.com/strg-at/charts/pull/11/files#diff-e3283a4797a9ca6d4c982c86d760d75fd357c9f34965bd8b77059b2992ec2ebeR46-R48

And check if we defined them https://github.com/strg-at/charts/pull/11/files#diff-6cd4859e0ab540a6579dd551e55ed00ed4d96bdacf6c7255706aad8234f62780R2-R10 @tyriis  can check if I defined them properly

Reply:
variables set as empty and check added to throw error in common template